### PR TITLE
Home page menu to have an option to access reports and dashboards

### PR DESCRIPTION
**Pull request by lucyjemutai**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes the ticket number in the format `OHRI-123 My PR title`.
- [ ] My work includes tests or is validated by existing tests.

## Summary

Home page menu to have an option to access reports and dashboards
## Screenshots

image
image

### CODE REVIEW

**Review by hadijahkyampeire — COMMENTED** [review]
@lucyjemutai I don't think you need to create a new esm, just have that component in the pmtct package. @CynthiaKamau PR in the rwanda esm will be good to look at because she did the exact thing

**Comment by lucyjemutai** [comment]
Thank you @hadijahkyampeire 
I will add the component in the pmtct package.

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "start:hct": "openmrs develop --backend https://openmrs-dev.globalhealthapp.net --sources packages/esm-hiv-care-treatment-app",
     "start:hp": "openmrs develop --backend https://openmrs-dev.globalhealthapp.net --sources packages/esm-hiv-prevention-app",
     "start:cervical-cancer": "openmrs develop --backend https://openmrs-dev.globalhealthapp.net --sources packages/esm-cervical-cancer-app",
-    "start:pmtct": "openmrs develop --backend https://ohri-namibia-dev.globalhealthapp.net --sources 'packages/esm-ohri-pmtct-app/'",
+    "start:pmtct": "openmrs develop --backend https://ohri-namibia-demo.globalhealthapp.net --sources 'packages/esm-ohri-pmtct-app/'",
```
Comment (hadijahkyampeire) [inline package.json:22]: Maybe you wanna put this back to dev

**Review by hadijahkyampeire — COMMENTED** [review]
Besides that comment, it looks good

**Review by hadijahkyampeire — APPROVED** [review]
LGTM

**Pull request merged**